package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
```

Comment:
Yes it works nicely for me as well. I think I used(required) `in-publish` inside the `jsreport-studio-build` before which was causing the mentioned travis error.  

Maybe the `npm` skips to run the prepublish script because `in-publish` is not there. This would be even more obscure than running it at all, but there must be some fun in the developer's life. :)

More likely, `npm` runs the prepublish just for the root packages during `npm` install. So running `npm install jsreport` won't run prepublish on `jsreport-images`, so it will work as expected.

To summary this, you can move it to devDependencies, I don't have an argument why it shouldn't be there.